utils/S3UtilsMongoClient.js

```diff
@@ -14,10 +14,40 @@ const __COUNT_ITEMS = 'countitems';
 
 
 class S3UtilsMongoClient extends MongoClientInterface {
+    /**
+     * Get the list of buckets and their location dates
+     * @param {object} log - Werelogs logger
+     * @returns {object} - Object with bucket names as keys
+     * and their creation dates as values
+     */
+    async _getUsersBucketCreationDates(log) {
+        let cursorUsersBucketCreationDates;
+        try {
+            cursorUsersBucketCreationDates = await this.getCollection(USERSBUCKET).find({}, {
+                projection: {
+                    'value.creationDate': 1,
+                },
+            });
+            const usersBucketCreationDatesArray = await cursorUsersBucketCreationDates.toArray();
+            return usersBucketCreationDatesArray
+                .reduce((map, obj) => ({ ...map, [obj._id]: obj.value.creationDate }), {});
+        } catch (err) {
+            log.error('Failed to read __usersbucket collection', {
```

Comment:
nit: any reason to catch the exception instead of letting it propagate up, like the other exceptions which may be generated from within `getObjectMDStats()` and get caught at the end of that function?